docs/extend/extend-apiml/onboard-plain-java-enabler.md

```diff
@@ -295,10 +296,12 @@ The onboarding configuration parameters are broken down into the following group
 
 *  **serviceIpAddress** (Optional)
 
-
     This parameter specifies the IP address of the service and can be provided by system administrator in externalized service configuration.
```

Comment:
"Specifies the service IP address and can be provided by a system administrator in the externalized service configuration.